bin/get-unused-components

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
```

Comment:
1. You are right. It is to be used in case we want thing a block is used and want to delete it safely. 
2. I used that script to see if I could remove the existing Banner Block as part of cleanup for this ticket. So it is related. Also, this code is copied from the Storyblok FAQ (link in the file), so this just adds the convenience if somebody else ends up in the same situation as me.

Reply:
I used this one when doing refactoring to Storyblok to see what blocks where used, so it doesn't delete anything but it's a pretty neat tool IMO :)

Reply:
Cool, I guess I mean we should still always check with the Storyblok editors if they plan on using a block that's currently not used before removing it :)

I still think it would be nice to have it in a separate branch and PR though, since it's not per se coupled with the changes here.